### PR TITLE
Added a new meme.

### PR DESCRIPTION
**Pull request by nxtCP**

### CODE REVIEW

File: cogs/Memes.py [hunk 1]
```diff
@@ -18,6 +18,12 @@ async def onlyfans(self, inter):
         await inter.send(
             "https://media.wired.com/photos/59548ac98e8cc150fa8ec379/master/w_2560%2Cc_limit/GettyImages-56196238.jpg"
         )
+    @commands.slash_command(name="/boners with lewis", description="Boners with lewis")
```
Comment (OpenSourceSimon) [inline cogs/Memes.py:21]:
```suggestion
    @commands.slash_command(name="/bonerswithlewis", description="Boners with lewis")
```

**Review by OpenSourceSimon — CHANGES_REQUESTED** [review]
Command with spaces don't work according to Levani

**Review by OpenSourceSimon — APPROVED** [review]
Looks good to me, @LevaniVashadze Can you review? When we're going to merge, make sure to squash merge because of the many commits.

**Comment by JasonLovesDoggo** [comment]
@nxtCP has it been tested?

**Comment by JasonLovesDoggo** [comment]
> @nxtCP has it been tested? 

Also change cmd name to lewisboner

File: cogs/Memes.py [hunk 2]
```diff
@@ -18,6 +18,12 @@ async def onlyfans(self, inter):
         await inter.send(
             "https://media.wired.com/photos/59548ac98e8cc150fa8ec379/master/w_2560%2Cc_limit/GettyImages-56196238.jpg"
         )
+    @commands.slash_command(name="/bonerswithlewis", description="Boners with lewis")
```
Comment (JasonLovesDoggo) [inline cogs/Memes.py:21]: Change cmd name to lewisboner other than that I approve

**Review by JasonLovesDoggo — CHANGES_REQUESTED** [review]

File: cogs/Memes.py (hunk 2 above)
Comment (OpenSourceSimon) [inline cogs/Memes.py:21]:
```suggestion
    @commands.slash_command(name="/lewisboner", description="Boners with lewis")
```

**Review by OpenSourceSimon — COMMENTED** [review]
Suggested change from Jason

**Comment by ImmaHarry** [comment]
Guys change the name, its NSFW. We cannot have a rule going against NSFW content & implement an NSFW feature at the same time, even though its a joke. Please use alternative wording for this command, I think it'd be okay to have it send that picture but the command name needs to be something else. End of discussion here.

**Comment by OpenSourceSimon** [comment]
> Guys change the name, its NSFW. We cannot have a rule going against NSFW content & implement an NSFW feature at the same time, even though its a joke. Please use alternative wording for this command, I think it'd be okay to have it send that picture but the command name needs to be something else. End of discussion here.
> 
> PS: It also creates a wrong impression for the time/new users on the server or of the bot.

Haven't seen it in that way, I totally agree with you

**Comment by LevaniVashadze** [comment]
> 

gotta agree on this one i also think we dont need to have such stuff in the bot since it just turns into a bit of spam and the onlyfans command was a fun meme and many of them ruins the purpose kind of

**Comment by LevaniVashadze** [comment]
also you could just go to #starboard or create a tag and would see that funny thing

**Comment by JasonLovesDoggo** [comment]
Can be implemented via tags directly, might add a admin only /tag a_alias add cmd that gives the tag a apex level name

**Pull request closed without merging**